src/stratigraphy/util/geometric_line_utilities.py

```diff
@@ -230,7 +232,7 @@ def merge_parallel_lines(lines: list[Line], tol: int = 8, angle_threshold: float
         if not merged:
             merged_lines.append(line)
     if merged_any:
-        return merge_parallel_lines(merged_lines, tol=tol)
+        return merge_parallel_lines(merged_lines, tol=tol, angle_threshold=angle_threshold)
```

Comment:
Ui, that's a good catch. You are right, I should be careful with default values.